susemanager-utils/susemanager-sls/modules/engines/mgr_events.py

```diff
@@ -194,60 +203,58 @@ def _get_queue(self, minion_id):
         return 0
 
     def trace_log(self):
-        log.trace("%s: queues sizes -> %s", __name__, self.counters)
-        log.trace("%s: tokens -> %s", __name__, self.tokens)
+        if self.counter or self._queue:
+            log.trace("queues sizes [%d] -> %s", len(self._queue), self.counters)
 
-    def _is_salt_mine_event(self, tag, data):
-        return fnmatch.fnmatch(tag, "salt/job/*/ret/*") and self._is_salt_mine_update(
-            data
-        )
-
-    def _is_salt_mine_update(self, data):
-        return data.get("fun") == "mine.update"
-
-    def _is_presence_ping(self, tag, data):
-        return (
-            fnmatch.fnmatch(tag, "salt/job/*/ret/*")
-            and self._is_test_ping(data)
-            and self._is_batch_mode(data)
-        )
-
-    def _is_test_ping(self, data):
-        return data.get("fun") == "test.ping"
-
-    def _is_batch_mode(self, data):
-        return data.get("metadata", {}).get("batch-mode")
+    def push_events_from_queue(self):
+        while self._queue and (
+            self.counter < self._commit_burst or self._commit_burst == 0
```

Comment:
I'm not getting what the goal/behavior of _commit_burst == 0? If it's 0 then we should always be getting events from the queue? Then when are they commited?